docs/users_guide/ocean/test_groups/baroclinic_gyre.rst

```diff
@@ -0,0 +1,180 @@
+.. _baroclinic_gyre:
+
+baroclinic_gyre
```

Comment:
This needs to be added to the index.rst for test groups, just as in the developer's guide.

Reply:
@xylar I included that change in docs/users_guide/ocean/test_groups/index.rst. Am I missing something?

Reply:
Sorry, I saw that and thought I deleted this comment. Please ignore it.